src/PageLayout/PageLayout.tsx

```diff
@@ -530,7 +564,7 @@ const Pane = React.forwardRef<HTMLDivElement, React.PropsWithChildren<PageLayout
     {
       'aria-label': label,
       'aria-labelledby': labelledBy,
-      position: responsivePosition = 'end',
+      position: responsivePosition = undefined,
```

Comment:
I think we should drop these changes for handling an `undefined` position prop. It adds extra complexity, and `"end"` is a reasonable default since everything should just flow as it would by source position alone.

The TS deprecation warning is enough to get the point across until `next-major` is merged in (which fully removes support for this).

Reply:
Agreed! I also think we won't need the other implementation changes in this minor PR since we are only introducing a deprecation notice. Am I missing anything? I pushed a [commit](https://github.com/primer/react/pull/3597/commits/f3f56d33376f1b770267405d669050ed8ae87e55) to take all them back and leave only the docs and deprecation notice but let me know what you think and if the other changes needed as well! @radglob

Reply:
Yeah, that sounds right to me.